pkg/actions/nodegroup/upgrade_test.go

```diff
@@ -26,6 +27,17 @@ import (
 )
 
 var _ = Describe("Upgrade", func() {
+	supportedVersions := api.SupportedVersions()
+	if len(supportedVersions) < 2 {
+		Fail("Upgrade test requires at least two supported EKS versions")
+	}
+
+	var (
+		eksVersion        = aws.String(supportedVersions[len(supportedVersions)-2])
+		latestEKSVersion  = supportedVersions[len(supportedVersions)-1]
+		eksReleaseVersion = aws.String(fmt.Sprintf("%s-20201212", *eksVersion))
+	)
```

Comment:
I think it would be better with ...

```
	var (
		latestEKSVersion  = supportedVersions[len(supportedVersions)-1]
		eksVersion        = aws.String(supportedVersions[len(supportedVersions)-2])
        eksVersionSuffix  = aws.String("20201212")
		eksReleaseVersion = aws.String(fmt.Sprintf("%s-%s", *eksVersion, *eksVersionSuffix))
	)
```

Reply:
There's no need to get the actual release date. These are unit tests. The mocks only need to ensure that the AMI lookup is performed with the correct parameters, the return value does not need to conform to any standard as it's only being used for comparison.